WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginUtils.java

```diff
@@ -18,6 +18,7 @@ public static boolean isPluginFeatureAvailable(SiteModel site) {
         // If the site has business plan we can do an Automated Transfer
```

Comment:
For the Jetpack check above, do we also need to verify the user is an admin?

Reply:
I already had that as a personal todo, but it's not a part of this PR.

Reply:
To elaborate a bit more, this admin check is for Automated Transfer, not for plugins feature.